junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -573,9 +573,11 @@ public static boolean isPackage(String packageName) {
 		Preconditions.notNull(clazz, "Class must not be null");
 		Preconditions.notNull(sortOrder, "MethodSortOrder must not be null");
 
-		// TODO [#333] Determine if we need to support bridged methods.
-
-		List<Method> localMethods = Arrays.asList(clazz.getDeclaredMethods());
+		// @formatter:off
+		List<Method> localMethods = Arrays.stream(clazz.getDeclaredMethods())
+				.filter(Method::isBridge) // [#333] don't collect bridge methods
```

Comment:
Now, the last test summery reads better:
```
:platform-tests:junitPlatformTest
Test run finished after 3391 ms
[        63 containers found      ]
[         0 containers skipped    ]
[        63 containers started    ]
[         0 containers aborted    ]
[        63 containers successful ]
[         0 containers failed     ]
[       502 tests found           ]
[         0 tests skipped         ]
[       502 tests started         ]
[         1 tests aborted         ]
[       501 tests successful      ]
[         0 tests failed          ]
```
Previous iteration:
```
:platform-tests:junitPlatformTest
Test run finished after 78 ms
[         4 containers found      ]
[         0 containers skipped    ]
[         4 containers started    ]
[         0 containers aborted    ]
[         4 containers successful ]
[         0 containers failed     ]
[         6 tests found           ]
[         0 tests skipped         ]
[         6 tests started         ]
[         0 tests aborted         ]
[         6 tests successful      ]
[         0 tests failed          ]
:platform-tests:test SKIPPED
:platform-tests:check
:platform-tests:build
BUILD SUCCESSFUL
Total time: 55.391 secs
```

Both build were *SUCCESSFUL*. 😲